Automattic/sensei
# Fix question editor in WordPress 5.5

Fixes #3509

### Changes proposed in this Pull Request

* Replace usages of `$.attr( "value"[, newValue] )` with `$.val( [newValue] )`.
* Replace usage of `$.load( fn() )` with `$.on( "load", fn() )`.
* Pre-commit hook also did some updating of formatting.

### Testing instructions

* Rebuild assets. Upgrade to WP 5.5-rc2 or install the JQuery Migrate Helper plugin and remove the migrate helper.
* Attempt to add a new question to a quiz.
* Verify grading/auto-grading isn't broken.
* Ensure it works with WP 5.4.x as well.
